hist/hist/inc/TH1.h

```diff
@@ -355,6 +355,7 @@ class TH1 : public TNamed, public TAttLine, public TAttFill, public TAttMarker {
            void     RecursiveRemove(TObject *obj) override;
    virtual void     Reset(Option_t *option = "");
    virtual void     ResetStats();
+           void     SaveAs(const char *filename, Option_t *option) const override;  // *MENU*
```

Comment:
The correct signature is:
```
void SaveAs(const char *filename  = "", Option_t *option = "") const override; // *MENU*
```

i.e. It must provide the same default as `TObject::SaveAs` that it overloads.